app/assets/javascripts/oxalis/workers/fetch_buffer_with_headers.worker.js

```diff
@@ -0,0 +1,26 @@
+// @flow
+
+import handleStatus from "libs/handle_http_status";
+import { expose } from "./comlink_wrapper";
+
+function fetchBufferWithHeaders(
+  url: RequestInfo,
+  options?: RequestOptions,
+): Promise<{ buffer: ArrayBuffer, headers: Object }> {
+  return fetch(url, options)
+    .then(handleStatus)
+    .then(async response => {
+      const buffer = await response.arrayBuffer();
+      const { headers } = response;
+      const headerObject = {};
+      for (const [key, value] of headers.entries()) {
```

Comment:
So the `headersTransferHandler` that is used to transfer Headers TO a webworker does not work when transferring Headers the other way? Or would it be possible to reuse it? :)

Reply:
Yeah, it's a bit shitty. I think the problem is that the header attribute is nested within the actual transferred data. I didn't feel like going into this further, but we should keep an eye on that in the feature.